src/Uno.UI/UI/Xaml/Media/VisualTreeHelper.cs

```diff
@@ -400,23 +407,15 @@ internal static UIElement ReplaceChild(UIElement view, int index, UIElement chil
 			throw new NotImplementedException("ReplaceChild not implemented on this platform.");
 		}
 
-		internal static IReadOnlyList<_View> ClearChildren(UIElement view)
```

Comment:
This method used to return `IReadOnlyList<_View>``, which makes the code there having to allocate a list. However, there is no caller using the return value :)